src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -832,8 +832,8 @@ protected function send_debug_email() {
 This debugging email is sent when you are using a development version of ClassicPress.
 
 If you think these failures might be due to a bug in ClassicPress, could you report it?
- * Open a thread in the support forums: https://wordpress.org/support/forum/alphabeta
- * Or, if you're comfortable writing a bug report: https://core.trac.wordpress.org/
+ * Open a thread in the support forums: https://docs.classicpress.net/faq-support/
+ * Or, if you're comfortable writing a bug report: https://docs.classicpress.net/faq-support/
```

Comment:
And then here either change the link (to Github issues) or the text (if changing the text I think it would be better to make 1 sentence from the two of them?

Reply:
Updated according to https://github.com/ClassicPress/ClassicPress/issues/198#issuecomment-435225554 . Happy to change it, if needed.

Reply:
We don't have support forums yet, so this doesn't really make sense.

It also shouldn't be the same link twice.  Per https://github.com/ClassicPress/ClassicPress/issues/198#issuecomment-435225554 you can use https://docs.classicpress.net/testing-classicpress/#reporting-bugs for the link to report bugs.